docs/_data/table/cn/user-configuration.yml

```diff
@@ -36,6 +36,8 @@ alluxio.user.file.write.avoid.eviction.policy.reserved.size.bytes:
   当用户选择LocalFirstAvoidEvictionPolicy作为写文件数据块的定位机制时,用户需要配置worker预留一些数据量来保证数据的存储，默认是0MB。
 alluxio.user.file.write.tier.default:
   数据块写入的默认存储层。可选值为整型数值。非负值代表从高层到底层的存储层（0代表第一层存储层，１代表第二层存储层，以此类推）。如果给定值大于存储层数量,这个数字代表最底层的存储层。负值代表从底层到高层的存储层（-1代表最底层存储层，-2代表次底层存储层，以此类推）如果给定值的绝对值大于存储层数量，这个数字代表最高层存储层。
+alluxio.user.file.passive.cache.enabled:
```

Comment:
move this up to be after `alluxio.user.file.metadata.load.type` so the keys are alphabetically sorted.

Reply:
Done. i've sort this file.